app/views/admin/project_logs/_log.html.slim

```diff
@@ -52,7 +52,7 @@
   - if log.summary
     .summary
       h5.sr-only = t('common.summary')
-      span = strip_tags(log.summary)
+      span = strip_tags(log.summary.to_s)
```

Comment:
What's this little sneaker doing stowing away in here? :D Can we clean up the commit history quickly so this isn't part of the PR? I can help with this if you like.